python/oneflow/utils/global_view/to_global.py

```diff
@@ -0,0 +1,179 @@
+        Usually used for making graph models's state dict global.
+
+    If you want to do the `split(0)` operation, but there are tensors that cannot be split by dim 0, then these tensors can specify sbp. 
+    It is worth noting that, for a tensor of shape `(1, n)`, you can specify SBP is `oneflow.sbp.split(1)`.
+    For example:
+
+    .. code-block:: python
+
+        flow.utils.to_global(state_dict, placement=placement, sbp=get_sbp)
+        # Defines a function to return the specified SBP.
+        def get_sbp(state_dict, tensor):
+            if tensor is state_dict["System-Train-TrainStep"]:
+                return BROADCAST
+            if tensor is state_dict["module_pipeline"]["m_stage3.linear.weight"]:
+                return flow.sbp.split(1)
+            if tensor is state_dict["module_pipeline"]["m_stage3.linear.bias"]:
+                return BROADCAST
+            return flow.sbp.split(0)
+
+    """
+    is_input_not_tensor_or_none = False
+    if (input is not None) and (not isinstance(input, (Tensor, dict, tuple, list))):
+        is_input_not_tensor_or_none = True
+
+    if (
+        (not is_input_not_tensor_or_none)
+        and (placement is not None)
+        and (not check_input_global(input))
+        and (not check_placement_on_all_ranks(placement))
+    ):
+        src_rank = placement.ranks.flat[0]
+        cur_rank = flow.env.get_rank()
+
+        if cur_rank == src_rank:
+            # Replace tensor(s) in the input with None, in order to reduce communication cost
+            if isinstance(input, Tensor) or input is None:
+                mapped_input_none = None
+            else:
+                input_tree_none = ArgsTree(input)
+
+                def leaf_fn_to_none(node):
+                    if isinstance(node, Tensor):
+                        return None
+                    else:
+                        warnings.warn(
+                            "Non-Tensor type: {} encountered, it will remain the same.".format(
+                                type(node)
+                            )
+                        )
+                        return node
+
+                mapped_input_none = input_tree_none.map_leaf(leaf_fn_to_none)
+
+            obj_input = pickle.dumps(mapped_input_none)
+            flow._oneflow_internal.cpu_broadcast(obj_input, src_rank)
+        else:
+            if cur_rank in placement.ranks:
+                # Participating in the broadcast process but retaining original value
+                flow._oneflow_internal.cpu_broadcast(None, src_rank)
+            else:
+                # The input of other ranks will be always overwritten no matter what is passed in
+                input = pickle.loads(
+                    flow._oneflow_internal.cpu_broadcast(None, src_rank)
+                )
+
+    is_tensor_transform_list = 0
+    if isinstance(input, Tensor):
+        input = [input]
+        is_tensor_transform_list = 1
```

Comment:
```suggestion
        is_tensor_transform_list = True
```

未来可以让 ArgsTree 直接支持 Tensor 吗，现在的处理方式感觉不太优雅